src/misc/cache/lv_cache.h

```diff
@@ -0,0 +1,67 @@
+/**
+* @file lv_cache.h
+*
+*/
+
+#ifndef LV_CACHE1_H
+#define LV_CACHE1_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "lv_cache_entry.h"
+#include "lv_cache_private.h"
+#include <stdbool.h>
+#include <stdlib.h>
+
+#include "_lv_cache_lru_rb.h"
+
+#include "lv_image_cache.h"
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ * GLOBAL PROTOTYPES
+ **********************/
+lv_cache_t * lv_cache_create(const lv_cache_class_t * cache_class,
+                             size_t node_size, size_t max_size,
+                             lv_cache_ops_t ops);
+void lv_cache_destroy(lv_cache_t * cache, void * user_data);
+
+lv_cache_entry_t * lv_cache_acquire(lv_cache_t * cache, const void * key, void * user_data);
```

Comment:
Please don't forget to add the API comments here.

Reply:
I'll be adding documentation to all the APIs I've added, and I'll be adding comments to some of the key code as well!